helper.psm1

```diff
@@ -102,15 +102,15 @@ Function findCompilerVersion
 
                 switch ($Matches['version'])
                 {
-                    2017 { VS2017 }
                     2019 { VS2019 }
-                    default { VS2017 }
+                    2022 { VS2022 }
+                    default { VS2019 }
                 }
             return
         }
     }
 
-    VS2017
+    VS2019
 }
```

Comment:
do we still need a default version? I'd consider arangodb/arangodb/VERSIONS not pinning it a bug from what we have now?
So, I'd raise an error if there is no match.

Reply:
That can be done via another specific VS PR: https://github.com/arangodb/oskar/pull/398.